clothcalc_placeholder.js

```diff
@@ -7799,13 +7807,26 @@
                         eval("MarketWindow.getClearName = " + save.MarketWindow)
                     }
                 };
+
+                _self.injectGetBids = function() {
+		    if (typeof save.MarketWindowTab == "undefined") {
```

Comment:
This way of injecting code into a function doesn't feel very reliable (`Function.prototype.toString`, `eval` and most importantly loosing access to scope*), nor readable. Maybe you just copied it from some of the code in this very file :D, but please have a look at `injectInventoryAddItemsPinItems` - this is the way I did it in one of my features. It's much more reliable and I also think nicer to read.

`*` loosing access to scope:
Suppose I have this code:
```
(function () {
    var x = 42;
    window.myFunction = function (a) { return a + x; }
})();
```
and I used your trick to inject some extra behavior to it, i.e. compiling a new function with almost the same body as `window.myFunction`. The newly compiled code wouldn't have access to the scope the previous function was created in and therefore `x` would be undefined. The way it's done in `injectInventoryAddItemsPinItems` doesn't suffer from this issue.